ui/src/app/views/workspace-edit-underserved-population/component.tsx

```diff
@@ -0,0 +1,320 @@
+      {id: UnderservedPopulationEnum.SEXUALORIENTATIONTWOSPIRIT, label: 'Two-Spirit'},
+      {
+        id: UnderservedPopulationEnum.SEXUALORIENTATIONFIGURINGOUTSEXUALITY,
+        label: 'Have not figured out or are in the process of figuring out their sexuality'
+      },
+      {
+        id: UnderservedPopulationEnum.SEXUALORIENTATIONMOSTLYSTRAIGHT,
+        label: 'Mostly straight, but sometimes attracted to people of their own sex'
+      },
+      {
+        id: UnderservedPopulationEnum.SEXUALORIENTATIONDOESNOTTHINKOFHAVINGSEXUALITY,
+        label: 'Does not think of themselves as having sexuality'
+      },
+      {
+        id: UnderservedPopulationEnum.SEXUALORIENTATIONDOESNOTUSELABELS,
+        label: 'Does not use labels to identify themselves'
+      },
+      {
+        id: UnderservedPopulationEnum.SEXUALORIENTATIONDOESNOTKNOWANSWER,
+        label: 'Does not know the answer'
+      }
+    ]
+  },
+  {
+    title: 'Geography',
+    categories: [
+      {
+        id: UnderservedPopulationEnum.GEOGRAPHYURBANCLUSTERS,
+        label: 'Urban clusters (2,500-50,000 people)'
+      },
+      {
+        id: UnderservedPopulationEnum.GEOGRAPHYRURAL,
+        label: 'Rural (All population, housing and territory not included within an urban area)'
+      }
+    ]
+  },
+  {
+    title: 'Disability',
+    categories: [
+      {id: UnderservedPopulationEnum.DISABILITYPHYSICAL, label: 'Physical Disability'},
+      {id: UnderservedPopulationEnum.DISABILITYMENTAL, label: 'Mental Disability'}
+    ]
+  },
+  {
+    title: 'Access to care',
+    categories: [
+      {
+        id: UnderservedPopulationEnum.ACCESSTOCARENOTPASTTWELVEMONTHS,
+        label: 'Have not had a clinic visit in the past 12 months'
+      },
+      {
+        id: UnderservedPopulationEnum.ACCESSTOCARECANNOTOBTAINORPAYFOR,
+        label: 'Cannot easily obtain or pay for medical care'
+      }
+    ]
+  },
+  {
+    title: 'Education',
+    categories: [
+      {
+        id: UnderservedPopulationEnum.EDUCATIONINCOMELESSTHANHIGHSCHOOLGRADUATE,
+        label: 'Less than a high school graduate'
+      },
+      {
+        id: UnderservedPopulationEnum.EDUCATIONINCOMELESSTHANTWENTYFIVETHOUSANDFORFOURPEOPLE,
+        label: 'Less than $25,000 for 4 people'
+      }
+    ]
+  }
+];
+
+export class WorkspaceUnderservedPopulation extends
+    React.Component<{value: Array<UnderservedPopulationEnum>, onChange: Function},
+    {show: boolean, value: Array<UnderservedPopulationEnum>}> {
+  constructor(props: any) {
+    super(props);
+    this.state = {show: false, value: !props.value ? [] : props.value};
+  }
+
+  flipShow() {
+    this.setState({
+      show: !this.state.show
+    });
+  }
+
+  focusCategoryChange(subCategory, value) {
+    if (!value) {
+      const index = this.state.value.findIndex(item => item === subCategory.id);
+      this.state.value.splice(index , 1);
+    } else if (value) {
+      this.state.value.push(subCategory);
```

Comment:
Mutating state in-place like this is not a good idea, since React has no way of knowing that the value has been modified. We'll be getting rid of this state anyway, but I wanted to mention it since it's an important principle to follow.